miden-crypto/src/merkle/mmr/proof.rs

```diff
@@ -7,6 +7,9 @@ use super::forest::Forest;
 
 #[derive(Debug, Clone, PartialEq)]
 #[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+/// Represents a proof for a leaf in a Merkle Mountain Range (MMR).
+///
+/// Contains the leaf's global position, the forest state, and the associated Merkle path.
```

Comment:
here and in other places too, we usually put attributes after comments, could we update this?